input/fsh/SD_Genomics.fsh

```diff
@@ -71,37 +71,43 @@ Description: "The result of a tumor marker test. Tumor marker tests are generall
+* extension contains RelatedCondition named relatedCondition 0..1 MS   // should this be 0..*?
+* extension[relatedCondition] ^short = "Condition associated with this test."
+* extension[relatedCondition] ^definition = "Associates the tumor marker test with a condition, if one exists. Condition can be given by a reference or a code. In the case of a screening test such as prostate-specific antigen (PSA), there may be no existing condition to reference."
 
-
-RuleSet: CancerRelatedSpecimenRules
+Profile: PatientSpecimen
+Parent: Specimen
+Id: mcode-patient-specimen
+Title:  "Patient Specimen Profile"
+Description:  "A specimen taken from a Patient. The profile includes extensions to capture a related condition, a more precise body site, and an identifier of source body structure at that site (for example, a tumor)."
 * ^extension[FMM].valueInteger = 1
-* type 1..1
-* subject only Reference(CancerPatient)
+* type from PatientSpecimenTypeVS (extensible)
+* subject only Reference(Patient)
 * subject ^definition = "The patient associated with this specimen."
 * collection.bodySite.extension contains
     BodyLocationQualifier named locationQualifier 0..* and
     LateralityQualifier named lateralityQualifier 0..1
-// It would be nice to reuse the existing condition-related extension (see Jira https://jira.hl7.org/projects/FHIR/issues/FHIR-31027) but it doesn't apply to Specimen
-* extension contains RelatedCondition named relatedCondition 0..1 MS
-* extension[relatedCondition].value[x] only Reference(PrimaryCancerCondition or SecondaryCancerCondition)
-* extension[relatedCondition] ^short = "Cancer condition associated with this specimen."
-* extension[relatedCondition] ^definition = "A reference that associates this specimen with a cancer condition."
-// No inherited MS
+// It would be nice to reuse the existing "condition-related" extension but it doesn't apply to Specimen (see Jira https://jira.hl7.org/projects/FHIR/issues/FHIR-31027) 
+* extension contains RelatedCondition named relatedCondition 0..1 MS  // should this be 0..*?
+* extension[relatedCondition] ^short = "Condition associated with this specimen."
+* extension[relatedCondition] ^definition = "Associates this specimen with a condition. For example, a sample of normal tissue can be associated with a cancer condition for tumor normal genomic testing, comparing tumor genome with normal tissue."
+* identifier ^slicing.discriminator.type = #pattern
+* identifier ^slicing.discriminator.path = "type"
+* identifier ^slicing.rules = #open
+* identifier ^slicing.description = "Slicing by identifier.type"
+* identifier contains bodyStructureIdentifier 0..*
+* identifier[bodyStructureIdentifier] only BodyStructureIdentifier
+* identifier[bodyStructureIdentifier] ^short = "Identifier to associate the specimen with an identified body structure, such as a tumor with a tumor identifier."
```

Comment:
SHouldn't identifier[bodyStructureIdentifier] be bound to a value set?

Reply:
The bodyStructureIdentifier is an arbitrary identifier that could be a number like "tumor 3" or "tumor C".